fossunited/api/proposal.py

```
@@ -265,3 +273,76 @@ def get_public_proposal_filters(
+    )
+
+    proposal_names = [p.get("name") for p in proposals] if proposals else []
+
+    anonymise = False
+    try:
+        cfp = (
+            frappe.db.get_value(EVENT_CFP, {"event": event}, ["anonymise_proposals"], as_dict=True)
+            or {}
+        )
+        anonymise = bool(cfp.get("anonymise_proposals"))
+    except Exception:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)